microservice-architecture/src/main/java/com/iluwatar/microservice/architecture/App.java

```diff
@@ -0,0 +1,18 @@
+package com.iluwatar.microservice.architecture;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+
```

Comment:
Here, above the main class, we need to describe the pattern and explain how the code implement it